nautobot/dcim/tests/test_filters.py

```diff
@@ -892,30 +866,42 @@ def test_has_prefixes(self):
             self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
 
     def test_vlan_groups(self):
-        vlan_groups = VLANGroup.objects.all()[:2]
+        vlan_groups = list(VLANGroup.objects.filter(site__isnull=False))[:2]
         params = {"vlan_groups": [vlan_groups[0].pk, vlan_groups[1].slug]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        self.assertQuerysetEqual(
+            self.filterset(params, self.queryset).qs, self.queryset.filter(vlan_groups__in=vlan_groups).distinct()
```

Comment:
Why was distinct required here?

Reply:
In general it seems that querysets using a reverse lookup, e.g. `queryset.filter(<related_name>__in=...)` can return duplicate entries. Not sure if every single case where I added `distinct()` is needed but at least some of them definitely were.